### PR TITLE
Milestone: Added logic to calculator

### PR DESCRIPTION
**Pull request by Lancelot-SO**
### The following updates were made: 

> Created a new directory called logic 

> Added both calculate.js and operate.js files to the logic directory. 

> Installed big.js 

> Implemented click event handlers using the logic from calculate.js and operate.js 

> Linters set up working properly

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,33 +1,64 @@
...
+            {' '}
+          </div>
+          <button onClick={this.onClickHandler} value="AC" className="key" type="button">AC</button>
+          <button onClick={this.onClickHandler} value="%" className="key" type="button">%</button>
+          <button onClick={this.onClickHandler} value="+/-" className="key" type="button">+/-</button>
+          <button onClick={this.onClickHandler} value="&divide;" className="key orange" type="button">&divide;</button>
+          <button onClick={this.onClickHandler} value="9" className="key" type="button">9</button>
+          <button onClick={this.onClickHandler} value="8" className="key" type="button">8</button>
+          <button onClick={this.onClickHandler} value="7" className="key" type="button">7</button>
+          <button onClick={this.onClickHandler} value="&times;" className="key orange" type="button">&times;</button>
```
Comment (Munsa1) [inline src/components/Calculator.js]: - [ ] When the user tries to multiply two numbers, clicking on the `=` sign redirects to a blank page. Kindly fix this issue

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Lancelot-SO, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Used the given math logic :heavy_check_mark: 
Used props and state :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Its good practice to write your commit messages in the imperative tense and to make them meaningful form here going forward. [Refer to this tutorial](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Whoistolu) [reply src/components/Calculator.js]: - Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Lancelot-SO 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ] [OPTIONAL] As recommended by the previous reviewer, it is a good practice to always write your commit messages in the imperative tense, to make them more meaningful and always start each commit message with a capital letter.
Your last commit reads "fixed error". It could have been better to use : "Fix multiplication errors" or "Resolve multiplication errors".

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**